packages/web-reporter-ui/src/sections/CPUReport.tsx

```diff
@@ -61,6 +61,9 @@ export const CPUReport = ({
         height={500}
         interval={POLLING_INTERVAL}
         series={totalCPUUsage}
+        annotationIntervalList={[
+          { y: 180, y2: 1000, color: "red", label: "Danger Zone" },
```

Comment:
can we make those constants? to make sure the big `useMemo` containing the options doesn't rerender

Also color wise, can we use:
        error: "#E62E2E",
        success: "#158000"
        
We can hardcode tbh if we don't have a fast way to reuse the theme